waspc/src/Wasp/Generator/WebAppGenerator/Test.hs

```diff
@@ -0,0 +1,15 @@
+module Wasp.Generator.WebAppGenerator.Test
+  ( testWebApp,
+  )
+where
+
+import StrongPath (Abs, Dir, Path', (</>))
+import Wasp.Generator.Common (ProjectRootDir)
+import qualified Wasp.Generator.Job as J
+import Wasp.Generator.Job.Process (runNodeCommandAsJob)
+import qualified Wasp.Generator.WebAppGenerator.Common as Common
+
+testWebApp :: [String] -> Path' Abs (Dir ProjectRootDir) -> J.Job
+testWebApp args projectDir = do
+  let webAppDir = projectDir </> Common.webAppRootDirInProjectRootDir
+  runNodeCommandAsJob webAppDir "npx" ("vitest" : args) J.WebApp
```

Comment:
I'm a bit confused if we are running the tests using the `npm run test` or `npx vitest`? Is one of those a leftover?